randovania/data/json_data/prime1.txt

```diff
@@ -998,23 +1003,13 @@ Asset id: 1282373491
 > Dock to Frost Cave Access; Heals? False
```

Comment:
Can't comment above since it's not changed but, Control Tower needs to be fixed. Getting the artifact needs Space Jump, Plasma, and a Missile only. Leaving from the artifact should be "and" not "or"